src/SDKs/ServiceBus/Management.ServiceBus/Generated/SdkInfo_ServiceBusManagementClient.cs

```diff
@@ -18,19 +19,19 @@ internal static partial class SdkInfo
           {
               return new Tuple<string, string, string>[]
               {
-                new Tuple<string, string, string>("ServiceBus", "Namespaces", "2018-01-01-preview"),
+                new Tuple<string, string, string>("ServiceBus", "DisasterRecoveryConfigs", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "EventHubs", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "MigrationConfigs", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Namespaces", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Operations", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "PremiumMessagingRegions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Queues", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Regions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Rules", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Subscriptions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Topics", "2017-04-01"),
               }.AsEnumerable();
           }
       }
-      // BEGIN: Code Generation Metadata Section
```

Comment:
Please pull down latest changes from remote and run `msbuild build.proj` and regenerate the code

Reply:
resolved